### PR TITLE
Improve dev docs

### PR DESCRIPTION
**Pull request by StachuDotNet**
WIP

### CODE REVIEW

File: docs/production/accounts.md [hunk 1]
```diff
@@ -4,64 +4,66 @@ List of accounts used by Dark as part of running Dark
...
-- ngrok.com
+- [auth0](https://manage.auth0.com/dashboard) user account management
+- [pusher.com](https://dashboard.pusher.com/apps) supports "real-time" pushes of information to the Editor
+- [algolia](https://www.algolia.com) search engine used to support [Darklang docs](https://docs.darklang.com)
+- [CloudFlare](https://cloudflare.com) slight usage for some DNS
+- [PositiveSSL](https://www.positivessl.com) manages certificates for darksa.com
+- [let's encrypt](https://letsencrypt.org) certificate management and custom domains
+  see [custom-domains](./custom-domains.md)
+- [brex](https://www.brex.com) how we pay for everything
+- [name.com](http://name.com) TODO what do we use this for?
```
Comment (StachuDotNet) [inline docs/production/accounts.md:47]: open question

File: docs/production/accounts.md [hunk 2]
```diff
@@ -4,64 +4,66 @@ List of accounts used by Dark as part of running Dark
...
+- [Honeycomb](https://honeycomb.io) for monitoring, production analysis, etc.
+  see [honeycomb.md](./honeycomb.md)
+- [Rollbar](https://rollbar.com) for recording, exploring, and resolving bugs
+- [PagerDuty](https://pagerduty.com) for alerting when things are broken
   - triggers from rollbar, pingdom and honeycomb
-- pingdom.com
-- sslmate.com
-- drlinkcheck.com
+- [Better UpTime](https://betteruptime.com) for alerting when things are broken (newer)
+- [Pingdom](https://pingdom.com) TODO: where do we use this?
```
Comment (StachuDotNet) [inline docs/production/accounts.md:60]: open question

File: docs/production/accounts.md [hunk 3]
```diff
@@ -4,64 +4,66 @@ List of accounts used by Dark as part of running Dark
...
+- [sslmate](https://sslmate.com) TODO: where do we use this?
```
Comment (StachuDotNet) [inline docs/production/accounts.md:61]: open question

File: docs/production/accounts.md [hunk 4]
```diff
@@ -4,64 +4,66 @@ List of accounts used by Dark as part of running Dark
...
+- [Dr. Link Check](https://drlinkcheck.com) TODO: where do we use this?
```
Comment (StachuDotNet) [inline docs/production/accounts.md:62]: open question

File: docs/production/accounts.md [hunk 5]
```diff
@@ -4,64 +4,66 @@ List of accounts used by Dark as part of running Dark
...
-# User comms, analytics
+# User communications, analytics
 
-- mailchimp
-- mandrill (automated emails)
-- heap analytics
-  - not really used
-- full story
-  - not really used
+- [Mailchimp](https://mailchimp.com) TODO: where do we use this?
```
Comment (StachuDotNet) [inline docs/production/accounts.md:66]: open question

**Review by pbiggar — APPROVED** [review]

File: docs/production/accounts.md (hunk 5 above)
Comment (pbiggar) [reply docs/production/accounts.md:66]: We add users to it when they sign up in ops-adduser canvas, and then we use it to email users when there are product updates.

File: docs/production/accounts.md (hunk 4 above)
Comment (pbiggar) [reply docs/production/accounts.md:62]: I get an email each week about broken links in the docs

File: docs/production/accounts.md (hunk 3 above)
Comment (pbiggar) [reply docs/production/accounts.md:61]: They email me if any of our SSL certs are misconfigured or expiring.

File: docs/production/accounts.md (hunk 2 above)
Comment (pbiggar) [reply docs/production/accounts.md:60]: We stopped using pingdom (moved to BetterUptime instead)

**Comment by StachuDotNet** [comment]
Thanks for the additional commentary - now in PR #4215

**Pull request merged**